src/actions/actions.jsx

```diff
@@ -1,5 +1,65 @@
-export function toggleSetting(name) {
-  return { type: 'TOGGLE_SETTING', name };
-}
+import $ from 'jquery';
 
-export default toggleSetting;
+// Account Page Component
+export const toggleSetting = name => ({
+  type: 'TOGGLE_SETTING',
+  name,
+});
+
+// Login Component
+export const updatePassword = password => ({
+  type: 'UPDATE_PASSWORD',
+  password,
+});
+
+export const updateUsername = username => ({
+  type: 'UPDATE_USERNAME',
+  username,
+});
+
+// Register Component
+export const updateNewUsername = username => ({
+  type: 'UPDATE_NEW_USERNAME',
+  username,
```

Comment:
I do agree with Logan, in that it would be a good idea to have keys that are more specific to what the component is.